system/hardware/tici/tests/test_power_draw.py

```diff
@@ -5,29 +5,33 @@
 import threading
 from dataclasses import dataclass
 from tabulate import tabulate
+from typing import List
 
 import cereal.messaging as messaging
+from cereal.services import service_list
 from system.hardware import HARDWARE, TICI
 from system.hardware.tici.power_monitor import get_power
 from selfdrive.manager.process_config import managed_processes
 from selfdrive.manager.manager import manager_cleanup
 
+SAMPLE_TIME = 8  # Sample power for 8 seconds
```

Comment:
bad practice to copy the value in the comment, easy to forget to update it
```suggestion
SAMPLE_TIME = 8   # seconds to sample power
```